v2/apiserver/internal/lib/queue/amqp/writer.go

```diff
@@ -0,0 +1,200 @@
+package amqp
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	amqp "github.com/Azure/go-amqp"
+	myamqp "github.com/brigadecore/brigade/v2/apiserver/internal/lib/amqp"
```

Comment:
Not particularly a fan of the `myamqp` import alias (but obviously, this is as nitty as is gets).

Reply:
I don't like it either. I'm extremely open to alternatives.